src/manifest.c

```diff
@@ -556,6 +574,10 @@ int load_manifests(int current, int version, char *component, struct file *file,
 		}
 	}
 
+	if (component) {
+		set_tracked_files(*manifest);
```

Comment:
If component == "MoM", this call is a bit weird, but because MoM manifests have an empty files list, should be okay.